js/main.js

```diff
@@ -0,0 +1,81 @@
+var $cityName = document.querySelector('.city-name');
+var $state = document.querySelector('.chose-state');
+var $submit = document.querySelector('.search-section');
+
+$submit.addEventListener('submit', submited);
+
+var $listingRow = document.querySelector('#listing');
+
+function submited(event) {
+  event.preventDefault();
+  var cityname = $cityName.value;
+  var state = $state.value;
+
+  for (var i = 0; i < cityname.length; i++) {
+    if (cityname[i] === ' ') {
+      cityname[i] = '%20';
+    }
+  }
```

Comment:
Does this do anything? If `cityname` is a string, then nothing will change, since strings are immutable. Also, please use camelCase for variable names